src/wallet/scriptpubkeyman.cpp

```diff
@@ -296,7 +296,22 @@ std::unique_ptr<SigningProvider> LegacyScriptPubKeyMan::GetSigningProvider(const
 
 bool LegacyScriptPubKeyMan::CanProvide(const CScript& script, SignatureData& sigdata)
```

Comment:
please explain in header what it `CanProvide`

Reply:
Done